en/docs/SetupAndInstall/UpgradingWSO2APIManager/upgrading-from-250-to-300.md

```diff
@@ -3,350 +3,1676 @@
-2.  Create a new database for the new statistics database and configure it in the `<APIM_ANALYTICS_2.6.0_HOME>/conf/worker/deployment.yaml` as follows:
-    In this example shows to configure MySQL database configurations.
+8. If you manually added any JAR files to the `<API-M_2.5.0_HOME>/repository/components/lib` directory, copy those and paste them in the `<API-M_3.0.0_HOME>/repository/components/lib` directory.
 
-    ``` java
-        - name: APIM_ANALYTICS_DB
-              description: "The datasource used for APIM statistics aggregated data."
-              jndiConfig:
-                name: jdbc/APIM_ANALYTICS_DB
-              definition:
-                type: RDBMS
-                configuration:
-                  jdbcUrl: 'jdbc:mysql://localhost:3306/spDatabase?autoReconnect=true'
-                  username: username
-                  password: password
-                  driverClassName: com.mysql.jdbc.Driver
-                  maxPoolSize: 50
-                  idleTimeout: 60000
-                  connectionTestQuery: SELECT 1
-                  validationTimeout: 30000
-                  isAutoCommit: false
-    ```
+9. Log4j version has been upgraded to log4j2 in API Manager 3.0.0. You will notice that there is a log4j2.properties file in the `<API-M_3.0.0_HOME>/repository/conf/` directory instead of the log4j.properties file. Follow [Upgrading to Log4j2](../UpgradingWSO2APIManager/upgrading-to-log4j2.md) to migrate your existing log4j configurations.
 
-3.  Copy the relevant JDBC driver to the `<APIM_ANALYTICS_2.6.0_HOME>/lib` folder.
-4.  Start the new API-M Analytics server.
+    !!! warning
+        Taking the log4j.properties file from your old WSO2 API-M Server and adding it to WSO2 API-M Server 3.0.0 will no longer work. Refer [Upgrading to Log4j2](../UpgradingWSO2APIManager/upgrading-to-log4j2.md) to see how to add a log appender or a logger to the log4j2.properties file.
 
-    ``` java
-            cd <APIM_ANALYTICS_2.6.0_HOME>/bin
-            sh worker.sh
-    ```
+    !!! note
+        Log4j2 has hot deployment support, and **Managing Logs** section has been removed from the Management Console. You can now use the log4j2.properties file to modify logging configurations without restarting the server.
 
-    When the WSO2 API-M Analytics 2.6.0 server starts, the new statistics tables will get generated in the database that you configured in **step 1.2.1 - (2.)** above.
+### Step 2 - Upgrade API Manager to 3.0.0
 
-##### Step 1.2.2 - Configure WSO2 API-M 2.6.0 for Analytics
+1.  Stop all WSO2 API Manager server instances that are running.
 
-Follow the instructions below to configure WSO2 API Manager for the WSO2 API-M Analytics migration in order to migrate the statistics related data.
+2.  Make sure you backed up all the databases and Synapse configurations as instructed in [step 1](#step-11-migrate-the-api-manager-configurations) of the previous section.
```

Comment:
Link should change as [step 1](#step-1-migrate-the-api-manager-configurations)

Reply:
Changed